osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1885,6 +1885,36 @@
 				"png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/meme/images/meme.png",
 				"svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/meme/images/meme.svg"
 			}
-		}
+		},    
+    {
+      "description": "Atolo the native token of Rizon Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/2716E3F2E146664BEFA9217F1A03BFCEDBCD5178B3C71CACB1A0D7584451D219",
+          "exponent": 0,
+          "aliases": [
+            "uatolo"
+          ]
+        },
+        {
+          "denom": "atolo",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/2716E3F2E146664BEFA9217F1A03BFCEDBCD5178B3C71CACB1A0D7584451D219",
+      "name": "Atolo",
+      "display": "atolo",
+      "symbol": "ATOLO",
+      "ibc": {
+        "source_channel": "channel-1",
+        "dst_channel": "channel-221",
+        "source_denom": "uatolo"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/rizon/images/atolo.png",
+        "svg": "https://raw.githubusercontent.com/rizon-world/rizon-assets/master/logo/symbol%26favicon.svg"
```

Comment:
Please change to "https://raw.githubusercontent.com/cosmos/chain-registry/master/rizon/images/atolo.svg" (while also having added the svg to the chain reg)